src/IBM.Cloud.SDK.Core/Http/HttpMediaType.cs

```diff
@@ -21,38 +21,39 @@ namespace IBM.Cloud.SDK.Core.Http
-        public const string APPLICATION_MS_WORD = "application/msword";
-        public const string APPLICATION_MS_WORD_DOCX = "application/vnd.openxmlformats-officedocument.wordprocessingml.document";
-        public const string APPLICATION_OCTET_STREAM = "application/octet-stream";
-        public const string APPLICATION_PDF = "application/pdf";
-        public const string APPLICATION_SVG_XML = "application/svg+xml";
-        public const string APPLICATION_XHTML_XML = "application/xhtml+xml";
-        public const string APPLICATION_ZIP = "application/zip";
-        public const string APPLICATION_XML = "application/xml";
-        public const string AUDIO_OGG = "audio/ogg; codecs=opus";
-        public const string AUDIO_OGG_VORBIS = "audio/ogg; codecs=vorbis";
-        public const string AUDIO_WAV = "audio/wav";
-        public const string AUDIO_PCM = "audio/l16";
-        public const string AUDIO_BASIC = "audio/basic";
-        public const string AUDIO_FLAC = "audio/flac";
-        public const string AUDIO_RAW = "audio/l16";
-        public static string createAudioRaw(int rate)
+        public const string ApplicationAtomXml = "application/atom+xml";
+        public const string ApplicationFormUrlEncoded = "application/x-www-form-urlencoded";
+        public const string ApplicationJson = "application/json";
+        public const string ApplicationMsWord = "application/msword";
+        public const string ApplicationMsWordDocx = "application/vnd.openxmlformats-officedocument.wordprocessingml.document";
+        public const string ApplicationOctetStream = "application/octet-stream";
+        public const string ApplicationPdf = "application/pdf";
+        public const string ApplicationSvgXml = "application/svg+xml";
+        public const string ApplicationXhtmlXml = "application/xhtml+xml";
+        public const string ApplicationZip = "application/zip";
+        public const string ApplicationXml = "application/xml";
+        public const string AudioOgg = "audio/ogg; codecs=opus";
+        public const string AudioOggVorbis = "audio/ogg; codecs=vorbis";
+        public const string AudioWav = "audio/wav";
+        public const string AudioPcm = "audio/l16";
+        public const string AudioBasic = "audio/basic";
+        public const string AudioFlac = "audio/flac";
+        public const string AudioRaw = "audio/l16";
+
+        public const string BinaryOctetStream = "binary/octet-stream";
+        public const string Json = ApplicationJson + "; charset=utf-8";
+        public const string MediaTypeWildcard = "*";
+        public const string MultipartFormData = "multipart/form-data";
+        public const string TextCsv = "text/csv";
+        public const string TextHtml = "text/html";
+        public const string TextPlain = "text/plain";
+        public const string Text = TextPlain + "; charset=utf-8";
+        public const string TextXml = "text/xml";
+        public const string Wildcard = "*/*";
```

Comment:
Realizing changing these will require a change in the generator for .NET ([example](https://github.ibm.com/CloudEngineering/openapi-sdkgen/blob/master/src/main/resources/ibm-dotnet-standard/service.mustache#L190)). Would this be considered breaking in the generator or not a big deal since it's not one of the tier 1 languages? @padamstx

Reply:
In general, a breaking change in a core library (resulting in a new major version of that core) does not necessarily mean there will be a corresponding breaking change in the generator, or that it will require a new  major  version of an SDK for that language.   When deciding these sorts of things, I like to focus on the SDK user (developer) and whether or not their EXISTING code that works with the current version of the SDK will continue to work (without source changes) in a new version of the SDK produced by (new generator + new core).

So, it's entirely possible that a breaking change of some sort is needed in the .NET core, and that will require a change in the .NET generator, but the resulting SDK code might not constitute a breaking change from the standpoint of the SDK user.

If the changes do in fact ripple all the way back to the SDK code and it represents a breaking change to SDK users, then I we'd also declare the generator change to be a breaking change and require a new major version of the generator.

However, having said all that... Nobody except your team is using the .NET generator currently, so I think it would be ok to deliver a breaking change in the 3.x .NET generator knowing that your team will not move up to that version of the generator until you're ready to deliver a new major version of your SDKs.

Clear as mud, right?